docs/frameworks/nvflare.md

```diff
@@ -0,0 +1,78 @@
+# FL using NVFlare in AzureML [Experimental]
+
+**Scenario** - This tutorial is an extension of the [pneumonia / radiography classification tutorial](../real-world-examples/pneumonia.md) using the NVFlare framework instead.  
+We run an NVFlare federated learning job to detect pneumonia from chest radiographs. We mimic a real-world FL scenario where 3 hospitals in 3 different regions want to collaborate on training a model to detect pneumonia from chest radiographs. The hospitals have their own data, and they want to train a model on all data without directly sharing data with each other, or with a central entity.  
+
+:warning: Experimental :warning: This tutorial is relying on some experiment code. It will work on our quickstart sandbox, but running it outside of this controlled setup might not work. We're actively working to release a more robust and generic solution.
+
+**Dataset** - The model is trained on the [Chest X-ray dataset](https://www.kaggle.com/datasets/paultimothymooney/chest-xray-pneumonia) from Kaggle. This example is adapted from [another FL solution](https://github.com/Azure/medical-imaging/tree/main/federated-learning) by Harmke Alkemade _et al._.
+
+## Install the required dependencies
+
+You'll need python to submit experiments to AzureML. You can install the required dependencies by running:
+
+```bash
+conda env create --file ./examples/pipelines/environment.yml
+conda activate fl_experiment_conda_env
+```
+
+Alternatively, you can just install the required dependencies:
+
+```bash
+python -m pip install -r ./examples/pipelines/requirements.txt
+```
+
+## Provision an FL sandbox workspace using vnet peering
+
+To run this example, you will need to provision an AzureML workspace ready for Federated Learning.
+
+**IMPORTANT**: Provision a [quickstart eyes-off/vnet-based setup](../quickstart.md) and select `applyVnetPeering=true` to apply peering to the orchestrator and silos vnet. Without this setting, this demo will NOT work.
+
+In the following tutorial, we will use the same names for the computes and datastores created by default during this quickstart.
+
+:notebook: take note of your workspace name, resource group and subscription id. You will need them to submit the experiment.
+
+## Create the datasets
+
+Follow the instructions in the [pneumonia real-world-example](../real-world-examples/pneumonia.md) to unpack the dataset: 1) add your kaggle credentials, 2) run a job to download and store the dataset in each silo.
+
+## Run the demo experiment
+
+1. If you are not using the quickstart setup, adjust the config file `project.yaml` in `examples/pipelines/pneumonia_nvflare/` to match your setup.
```

Comment:
Even if using the quickstart setup, the compute names and datastores don't match so this won't run

Reply:
Tests by Amit confirmed the job is working seamlessly, I'll mark this one as resolved.